ports/stm32f4/common-hal/busio/SPI.c

```diff
@@ -169,17 +219,17 @@ void common_hal_busio_spi_construct(busio_spi_obj_t *self,
     self->bits = 8;
 
     claim_pin(sck);
-    claim_pin(mosi);
-    claim_pin(miso);
+    if (self->mosi != NULL) claim_pin(mosi);
```

Comment:
In the past, @tannewt has asked me to {}-ify even single statement conditionals

Reply:
I hear ya but it pains me for these one liners. @tannewt are you hard and fast on this one?

Reply:
I am hard and fast on this, because of stuff like: https://dwheeler.com/essays/apple-goto-fail.html. In the long run we will clang-format the code anyway.

Reply:
Ok, I'll add them in.

Reply:
Yup, {} is the way to go. I try to be relaxed on style because I intend to turn clang-format on at some point. {} around single statements is actually one of my main reasons to do it. Unfortunately, clang-format can't actually do it. clang-tidy can but it requires a bit more tooling.